packages/beacon-node/src/chain/prepareNextSlot.ts

```diff
@@ -169,7 +193,7 @@ export class PrepareNextSlotScheduler {
             fromHex(headRoot),
```

Comment:
use `proposerHeadRoot` in case proposer boost reorg?

```
In case should_override_forkchoice_update returns True, a node SHOULD instead call notify_forkchoice_updated with parameters appropriate for building upon the parent block
```

Reply:
> use `proposerHeadRoot` in case proposer boost reorg?
> 
> ```
> In case should_override_forkchoice_update returns True, a node SHOULD instead call notify_forkchoice_updated with parameters appropriate for building upon the parent block
> ```

Yes you are right. Thanks for catching this🙏